cmd/describe/cluster/cmd.go

```diff
@@ -18,6 +18,7 @@ package cluster
 
 import (
 	"fmt"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
```

Comment:
please move this import below with the rest of the third-party ones

Reply:
updated